vuepress/docs/v6.3.2/docs/getting-started/README.md

```diff
@@ -4,17 +4,43 @@ sidebarDepth: 2
 
 # Getting Started with Entando
 
-::: warning
+::: warning 
 **Mac:** Entando 6 is not currently compatible with the Apple M1 ARM64 architecture found in some newer Macs.
 :::
 
-There are two options for getting started with the Entando platform.
-1. [Automatically install Entando via the Entando command-line interface (CLI)](#automatic-install). This is the fastest way to initialize an Entando application in Kubernetes. A CLI install defaults certain settings and prepares a local developer environment.
-2. [Manually install Entando step by step](#manual-install). This permits configuration of either a shared cluster or a local developer environment. A manual install is appropriate when CLI defaults don't meet specific needs, or to customize the deployment.
+## Try Entando
+
+Set up Entando using two simple steps or install manually for your specific needs. Then try the platform with a simple widget or walk through the **Welcome Wizard** in our AppBuilder. For a more in-depth experience, check out our **Banking demo**.
+
+* [Quick Local Install](https://dev.entando.org/v6.3.2/docs/getting-started/#automatic-install)	
+The fastest way to start up Entando in Kubernetes.
+
+* [Manual Install](https://dev.entando.org/v6.3.2/docs/getting-started/#manual-insta)	
+Useful if you want a shared cluster or customize. 
+
+  Once you're setup, check out:
+    * [Build a simple Bundle and create an App](https://dev.entando.org/v6.3.2/tutorials/ecr/publish-simple-bundle.html)
+    * [Welcome Wizard](https://dev.entando.org/v6.3.2/docs/getting-started/welcome-wizard.html)
+    * [Install our Banking Demo](https://dev.entando.org/v6.3.2/tutorials/samples/install-standard-demo.html)
+
+## Learn about Entando
+
+Learn about what Entando can do top down or explore the elements that make automating containers with Kubernetes easier.
+
+* [Introduction to Entando ](https://dev.entando.org/v6.3.2/docs/)	
```

Comment:
Style guide-wise, all links should be relative links, meaning do not include the full URL. I think there are notes on that in the README. If you include the full URL, these links will not point to the correct location in different environments, e.g. local, staging, etc.

Reply:
@nshaw please give an example of how to turn a full url link into a relative one?

Reply:
Relative means the path to the new page is relative to the location of the current page. So ./abc/def.md would check in the current location for an abc folder and then display the def.md in that folder. You can also use .. to go up a level (or more levels with ../.. or ../../.., etc.) , e.g. ../docs/getting-started/ to go up one level then into docs and getting started. You can also skip the path altogether and just use #section-name if you're referencing a different section in the current page. Unless you're pointing to an external web site we want to avoid anything with dev.entando.org in it and even v6.3.2, for maintenance reasons.